core/storage/user/gae_models.py

```diff
@@ -2362,11 +2361,15 @@ class PendingDeletionRequestModel(base_models.BaseModel):
     email = ndb.StringProperty(required=True)
     # Whether the deletion is completed.
     deletion_complete = ndb.BooleanProperty(default=False, indexed=True)
+
     # IDs of all the private explorations created by this user.
     exploration_ids = ndb.StringProperty(repeated=True, indexed=True)
     # IDs of all the private collections created by this user.
     collection_ids = ndb.StringProperty(repeated=True, indexed=True)
 
+    # Story to pseudonymous user ID mapping.
+    story_mappings = ndb.JsonProperty(default={})
```

Comment:
In the comment, give more thorough details (including typeinfo) about the internal structure of this field. (I'm a bit uncertain about its current naming but I can't really decide on that until I understand its structure.)